.github/workflows/playwright.yml

```
@@ -58,6 +58,17 @@ jobs:
           # prevent warnings if shard does not update any screenshots
           if-no-files-found: ignore
 
+      - name: Upload Reports
+        uses: actions/upload-artifact@v4
+        if: ${{ !cancelled() }}
+        with:
+          name: blob-report-${{ matrix.shard }}
+          path: |
+            ./packages/**/blob-report
+            ./apps/**/blob-report
+            !./**/node_modules/**
```

```suggestion
          path: |
            ./packages/*/blob-report
            ./apps/*/blob-report
```

Isn't this the same?